packages/utils-compat-data/src/browsers.ts

```diff
@@ -247,5 +248,15 @@ export const getUnsupportedBrowsers = (feature: Identifier | undefined, prefix:
         }
     }
 
-    return unsupported.length ? { browsers: unsupported, details } : null;
+    if (unsupported.length === 0) {
+        return null;
+    }
+
+    let mdnUrl = feature.__compat.mdn_url;
+
+    if (!mdnUrl && parent && parent.__compat) {
```

Comment:
Not sure if there's a concrete example of this in the current data, but couldn't the documentation be more than one level up? 

If so, it may be more future-proof to pass a list of ancestors instead of a single parent and loop backwards through those hunting for an `mdn_url`. 

That said, if we can't find any current examples, I don't think it's worth significant additional work to support this now.

Reply:
As we talk before, in the current data, I didn't find any more than one level up